chaosgcp/dns/actions.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from typing import Any, Dict
+
+from chaoslib.types import Configuration, Secrets
+
+from chaosgcp import client
+
+__all__ = ["update_service"]
+
+
+def update_service(
```

Comment:
Please rename this action as it doesn't have a clear purpose

Reply:
In fact, should we have more specialized actions like `add_A_record`, `set_cname`...

Reply:
yes I can write them later